qm.8.md

```diff
@@ -0,0 +1,121 @@
+% QM 8
+
+## NAME
+QM - Setup Containerized environment for running Functional Safety QM (Quality Management) software.
+
+## SYNOPSIS
+This package allow users to setup an environment which prevents applications
+and container tools from interfering with other all other processes on the
+system.
+
+The QM runs its own version of systemd and Podman to isolate not only the
+applications and containers launched by systemd and Podman but systemd and
+Podman themselves.
```

Comment:
```suggestion
Podman themselves. In other words, the systemd and Podman launched within the QM environment cannot affect the systemd or Podman processes running outside of that environment.
```

I was thinking to add in the example of Podman holding on a lock in QM won't affect Podman running outside of it, but it starts to feel a bit redundant. Or we drop the sentence I propose above and use that concrete example instead?